<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowUi</class>
 <widget class="QMainWindow" name="MainWindowUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GDB ImageWatch</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0">
          <property name="spacing">
           <number>1</number>
          </property>
          <item>
           <widget class="SymbolSearchInput" name="symbolList">
            <property name="minimumSize">
             <size>
              <width>210</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="statusTip">
             <string/>
            </property>
            <property name="accessibleDescription">
             <string/>
            </property>
            <property name="placeholderText">
             <string>add symbols (ctrl+k)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="imageList">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>210</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">padding: 0px;</string>
            </property>
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAsNeeded</enum>
            </property>
            <property name="sizeAdjustPolicy">
             <enum>QAbstractScrollArea::AdjustIgnored</enum>
            </property>
            <property name="dragDropMode">
             <enum>QAbstractItemView::InternalMove</enum>
            </property>
            <property name="iconSize">
             <size>
              <width>100</width>
              <height>100</height>
             </size>
            </property>
            <property name="resizeMode">
             <enum>QListView::Adjust</enum>
            </property>
            <property name="spacing">
             <number>4</number>
            </property>
            <property name="viewMode">
             <enum>QListView::ListMode</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="frame_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="rightPanelLayout">
          <item>
           <widget class="QWidget" name="toolbar" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <widget class="QToolButton" name="acToggle">
             <property name="geometry">
              <rect>
               <x>49</x>
               <y>11</y>
               <width>35</width>
               <height>24</height>
              </rect>
             </property>
             <property name="toolTip">
              <string>Toggle Auto Contrast</string>
             </property>
             <property name="accessibleDescription">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../resources/resources.qrc">
               <normaloff>:/resources/icons/contrast.png</normaloff>:/resources/icons/contrast.png</iconset>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
            <widget class="QToolButton" name="acEdit">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>11</y>
               <width>35</width>
               <height>24</height>
              </rect>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="toolTip">
              <string>Toggle min and max intensity editor</string>
             </property>
             <property name="icon">
              <iconset resource="../resources/resources.qrc">
               <normaloff>:/resources/icons/arrow-down-b.png</normaloff>:/resources/icons/arrow-down-b.png</iconset>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
            <widget class="QToolButton" name="reposition_buffer">
             <property name="geometry">
              <rect>
               <x>88</x>
               <y>11</y>
               <width>35</width>
               <height>24</height>
              </rect>
             </property>
             <property name="toolTip">
              <string extracomment="Reposition buffer on center">Reposition buffer to fit window</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../resources/resources.qrc">
               <normaloff>:/resources/icons/arrow-shrink.png</normaloff>:/resources/icons/arrow-shrink.png</iconset>
             </property>
            </widget>
            <widget class="QToolButton" name="linkViewsToggle">
             <property name="geometry">
              <rect>
               <x>127</x>
               <y>11</y>
               <width>35</width>
               <height>24</height>
              </rect>
             </property>
             <property name="toolTip">
              <string>Move all buffers simultaneously</string>
             </property>
             <property name="icon">
              <iconset resource="../resources/resources.qrc">
               <normaloff>:/resources/icons/link.png</normaloff>:/resources/icons/link.png</iconset>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
            <widget class="QToolButton" name="rotate_90_cw">
             <property name="geometry">
              <rect>
               <x>166</x>
               <y>11</y>
               <width>35</width>
               <height>24</height>
              </rect>
             </property>
             <property name="toolTip">
              <string>Rotate 90° clockwise</string>
             </property>
             <property name="icon">
              <iconset resource="../resources/resources.qrc">
               <normaloff>:/resources/icons/rotate-cw.png</normaloff>:/resources/icons/rotate-cw.png</iconset>
             </property>
            </widget>
            <widget class="QToolButton" name="rotate_90_ccw">
             <property name="geometry">
              <rect>
               <x>205</x>
               <y>11</y>
               <width>35</width>
               <height>24</height>
              </rect>
             </property>
             <property name="toolTip">
              <string>Rotate 90° counterclockwise</string>
             </property>
             <property name="icon">
              <iconset resource="../resources/resources.qrc">
               <normaloff>:/resources/icons/rotate-ccw.png</normaloff>:/resources/icons/rotate-ccw.png</iconset>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="minMaxEditor" native="true">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>85</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>85</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <widget class="QWidget" name="layoutWidget">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>540</width>
               <height>81</height>
              </rect>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="label_min">
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>(min)</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="label_red_min">
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>Red</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QLineEdit" name="ac_red_min">
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
               </widget>
              </item>
              <item row="0" column="3">
               <widget class="QLabel" name="label_green_min">
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>Green</string>
                </property>
               </widget>
              </item>
              <item row="0" column="4">
               <widget class="QLineEdit" name="ac_green_min">
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
               </widget>
              </item>
              <item row="0" column="5">
               <widget class="QLabel" name="label_blue_min">
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>Blue</string>
                </property>
               </widget>
              </item>
              <item row="0" column="6">
               <widget class="QLineEdit" name="ac_blue_min">
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_max">
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>(max)</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLabel" name="label_red_max">
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>Red</string>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QLineEdit" name="ac_red_max">
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
               </widget>
              </item>
              <item row="1" column="3">
               <widget class="QLabel" name="label_green_max">
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>Green</string>
                </property>
               </widget>
              </item>
              <item row="1" column="6">
               <widget class="QLineEdit" name="ac_blue_max">
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
               </widget>
              </item>
              <item row="1" column="4">
               <widget class="QLineEdit" name="ac_green_max">
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
               </widget>
              </item>
              <item row="1" column="5">
               <widget class="QLabel" name="label_blue_max">
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="text">
                 <string>Blue</string>
                </property>
               </widget>
              </item>
              <item row="0" column="9">
               <widget class="QToolButton" name="ac_reset_min">
                <property name="text">
                 <string>Reset</string>
                </property>
               </widget>
              </item>
              <item row="1" column="9">
               <widget class="QToolButton" name="ac_reset_max">
                <property name="text">
                 <string>Reset</string>
                </property>
               </widget>
              </item>
              <item row="1" column="7">
               <widget class="QLabel" name="label_alpha_max">
                <property name="text">
                 <string>Alpha</string>
                </property>
               </widget>
              </item>
              <item row="0" column="7">
               <widget class="QLabel" name="label_alpha_min">
                <property name="text">
                 <string>Alpha</string>
                </property>
               </widget>
              </item>
              <item row="0" column="8">
               <widget class="QLineEdit" name="ac_alpha_min"/>
              </item>
              <item row="1" column="8">
               <widget class="QLineEdit" name="ac_alpha_max"/>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="GLCanvas" name="bufferPreview" native="true">
            <property name="mouseTracking">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GLCanvas</class>
   <extends>QWidget</extends>
   <header location="global">src/ui/gl_canvas.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SymbolSearchInput</class>
   <extends>QLineEdit</extends>
   <header>src/ui/symbol_search_input.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>acEdit</sender>
   <signal>toggled(bool)</signal>
   <receiver>minMaxEditor</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>507</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot1()</slot>
 </slots>
</ui>
